### PR TITLE
[137] Product Page using HTML

### PR DESCRIPTION
**Pull request by DevOge**
IMG_6466

### CODE REVIEW

File: submissions/OgeEjim/HTML/HOME PAGE FOR MORE CAKES.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+                            <li>Reduce bad cholesterol</li>
+                            <li>Regulate blood sugar level</li>
+                            <li>Rejuvenate skin and</li>
+                            <li>Prevent neurodegenerative disease</li>
+                        </ul>
+                        This rich and tasty Cinnamon cakes are perfect for both the young and old. You can snack on this cake guilt free!
+                    </p>
+                </article>
+            </section>
+        </header>
```
Comment (Ireti203) [inline submissions/OgeEjim/HTML/HOME PAGE FOR MORE CAKES.html:28]: You dont need to have two header tags.

**Review by Ireti203 — CHANGES_REQUESTED** [review]

File: submissions/OgeEjim/HTML/HOME PAGE FOR MORE CAKES.html (hunk 1 above)
Comment (DevOge) [reply submissions/OgeEjim/HTML/HOME PAGE FOR MORE CAKES.html]: Ok ma, I've removed the first header.

File: submissions/OgeEjim/HTML/HOME PAGE FOR MORE CAKES.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+                <li>Oatmeal cupcakes</li>
+                <li>Banana cupcakes</li>
+                <li>Apple cupcakes</li>
+                <li>Honey cupcakes</li>
+             </ol>
+        </p>
+    </body>
+    <footer>
+        <p>
+
```
Comment (Ireti203) [inline submissions/OgeEjim/HTML/HOME PAGE FOR MORE CAKES.html:39]: Remove whitespace

File: submissions/OgeEjim/HTML/HOME PAGE FOR MORE CAKES.html [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+             </ol>
+        </p>
+    </body>
+    <footer>
+        <p>
+        
+           <blockquote><b>"Let food be thy medicine, and medicine thy food"~</b>
+            <em> Aristotle</em>
+           </blockquote>
+
```
Comment (Ireti203) [inline submissions/OgeEjim/HTML/HOME PAGE FOR MORE CAKES.html:43]: Same here

**Review by Ireti203 — CHANGES_REQUESTED** [review]
Remove the spaces in your file names

File: submissions/OgeEjim/HTML/HOME PAGE FOR MORE CAKES.html (hunk 3 above)
Comment (DevOge) [reply submissions/OgeEjim/HTML/HOME PAGE FOR MORE CAKES.html]: Ma, I've removed the spaces between names... I'm not sure my attempt at removing whitespaces worked … I don't see a difference in the web page

File: submissions/OgeEjim/HTML/HOMEPAGEFORMORECAKES.html [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+                <li>Oatmeal cupcakes</li>
+                <li>Banana cupcakes</li>
+                <li>Apple cupcakes</li>
+                <li>Honey cupcakes</li>
+             </ol>
+        </p>
+    </body>
+    <footer>
+        <p>
+
```
Comment (Ireti203) [inline submissions/OgeEjim/HTML/HOMEPAGEFORMORECAKES.html:39]: Remove whitespace

File: submissions/OgeEjim/HTML/HOMEPAGEFORMORECAKES.html [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+             </ol>
+        </p>
+    </body>
+    <footer>
+        <p>
+        
+           <blockquote><b>"Let food be thy medicine, and medicine thy food"~</b>
+            <em> Aristotle</em>
+           </blockquote>
+
```
Comment (Ireti203) [inline submissions/OgeEjim/HTML/HOMEPAGEFORMORECAKES.html:43]: Remove whitespace

File: submissions/OgeEjim/HTML/HOMEPAGEFORMORECAKES.html [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+        </p>
+    </body>
+    <footer>
+        <p>
+        
+           <blockquote><b>"Let food be thy medicine, and medicine thy food"~</b>
+            <em> Aristotle</em>
+           </blockquote>
+            
+        </P>
```
Comment (Ireti203) [inline submissions/OgeEjim/HTML/HOMEPAGEFORMORECAKES.html:44]: Use the proper closing p tag

**Review by Ireti203 — CHANGES_REQUESTED** [review]

**Review by Ireti203 — CHANGES_REQUESTED** [review]
Kindly use small letters or camelCasing to name your html file

**Comment by DevOge** [comment]
Done ma,
Good Morning

**Review by Ireti203 — APPROVED** [review]

**Pull request merged**